packages/blockchain-wallet-v4-frontend/src/config/index.js

```diff
@@ -1,13 +1,15 @@
 import Bitcoin from 'bitcoinjs-lib'
 import BitcoinCash from 'bitcoinforksjs-lib'
 
+const isTestnet = false
```

Comment:
I think a better way to do this is to determine the network through wallet-options. https://github.com/blockchain/wallet-options/blob/master/prod/wallet-options-v4.json#L30. So I will probably end up removing NETWORK related things from this config